.github/workflows/codecov.yml

```diff
@@ -6,33 +6,21 @@ on:
     branches: [master]
 
 jobs:
-  coverage:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
+  codecov:
+    runs-on: ubuntu-latest
     env:
-      OS: ${{ matrix.os }}
-      PYTHON: '3.7'
-
+      PYTHON: '3.9'
     steps:
-    - uses: actions/checkout@master
-
-    - name: Setup Python
-      uses: actions/setup-python@master
-      with:
-        python-version: 3.7
-
-    - name: Install fmu-tools and test dependencies
-      run: |
-        pip install pip -U
-        pip install .[tests]
-
-    - name: Generate coverage report
-      run: |
-        git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
-        pytest tests --disable-warnings --cov=fmu --cov-report=xml
-
-    - name: Upload coverage to Codecov
-      run: |
-        bash <(curl -s https://codecov.io/bash)
+      - uses: actions/checkout@v1
```

Comment:
Is there a reason for pinning to `checkout@v1` rather than `v3`, which is the latest big version I believe?

Reply:
Not any particular; `v1` works so I haven't tested `v3`